deployment/postgres-init-db/sql/tables/scheduler/scheduling_run/scheduling_goal_analysis_created_activities.sql

```diff
@@ -1,19 +1,18 @@
 create table scheduler.scheduling_goal_analysis_created_activities (
   analysis_id integer not null,
-  goal_id integer not null,
-  goal_revision integer not null,
+  goal_invocation_id integer not null,
   activity_id integer not null,
 
   constraint created_activities_primary_key
-    primary key (analysis_id, goal_id, goal_revision, activity_id),
+    primary key (analysis_id, goal_invocation_id, activity_id),
   constraint created_activities_references_scheduling_request
     foreign key (analysis_id)
-      references scheduler.scheduling_request (analysis_id)
+      references scheduler.scheduling_request
```

Comment:
No real reason, just following the pattern for the constraint below.

If I had to justify the style choice I'd say omitting this explicit column reference means PG links to the PK for that table, instead of just _a_ column, which I think represents our intentions better.

Reply:
While I support the change from a consistency standpoint and will resolve the comment, I want to mention that Postgres makes the exact same key regardless of if `analysis_id` is specified, it just defaults to the table's pk, [as mentioned in the docs](https://www.postgresql.org/docs/current/ddl-constraints.html#DDL-CONSTRAINTS-FK).

Reply:
Yep that's exactly what I stated above